__main__.py

```diff
@@ -19,11 +19,10 @@
 
 if __name__ == "__main__":
     RUN_PATH = os.path.join(os.getcwd(), 'PeekingDuck/run_config.yml')
-    CUSTOM_NODE_PATH = os.path.join(os.getcwd(), 'PeekingDuck/custom_nodes')
 
     setup_logger()
     logger = logging.getLogger(__name__)
     logger.info("Run path: %s", RUN_PATH)
 
-    runner = pkd.Runner(RUN_PATH, CUSTOM_NODE_PATH)
+    runner = pkd.Runner(RUN_PATH, "PeekingDuck")
```

Comment:
if custom folder path is taken from `cli` instead, may want to call `cli` here instead of skipping it and calling `runner` directly.

Reply:
Understood that the cli is just for creating the folder not to control the input for `custom_folder_path`. Above comment is not relevant now.